example/src/main/java/com/vimeo/android/networking/example/vimeonetworking/TestAccountStore.java

```diff
@@ -33,24 +35,25 @@ public TestAccountStore(Context context) {
 //        mAccountManager = AccountManager.get(context);
     }
 
+    @Nullable
     @Override
     public VimeoAccount loadAccount() {
         return AccountPreferenceManager.getClientAccount();
     }
 
-    @Deprecated
     @Override
-    public void saveAccount(VimeoAccount vimeoAccount, String email, String password) {
-        saveAccount(vimeoAccount, email);
+    public void saveNonUserAccount(@NonNull VimeoAccount vimeoAccount) {
```

Comment:
Call it `saveClientCredentialsAccount`? This seems to be more specific. Unless I'm missing another form of "non-user" account.